ui/component/or-asset-tree/src/index.ts

```diff
@@ -269,6 +271,13 @@ export class OrAssetTree extends subscribe(manager)(LitElement) {
     protected _expandedNodes: UiAssetTreeNode[] = [];
     protected _initCallback?: EventCallback;
 
+    @property({attribute: false})
+    protected _filterValue: string = '';
+    @property({type: Boolean})
```

Comment:
For fields that should cause a re-render but are protected then the `@state` directive should be used rather than the `@property`...this was introduced in a more recent lit version so lots of our codebase needs updating but good to adopt this for new code:

https://lit.dev/docs/components/decorators/#built-in-decorators

Reply:
Ok